helpers/dbload.ts

```diff
@@ -17,10 +18,10 @@ const sequelize = new Sequelize(
     port: parseInt(process.env.DB_PORT as string),
     dialect: 'postgres',
     pool: {
-      max: 5,
+      max: process.env.VERCEL_REGION ? 1 : 5,
```

Comment:
There's a more basic env var that vercel provides, its value will be 1 if it's running on a vercel server.

```suggestion
      max: process.env.VERCEL ? 1 : 5,
```

Reply:
I need to separate normal vercel environment from serverless. With `process.env.VERCEL` our production server will always use on connection which is bad and will cause problems.

Reply:
What problems?

Reply:
Additional delay. If server gets two queries simultaneously one needs to wait.